the-collab-lab/tcl-65-smart-shopping-list
# Draft - fixing the background image in the live build

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

A place to test solutions to the background image not acting as expected in the live build. This demo should show the same behavior.

Currently commented out the background image in the index.html root file.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓    | :bug: Bug fix              |
|     | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before
The background image does not appear, only the background image appears, and the background image flashes.

### After



## Testing Steps / QA Criteria
1. Load the demo and see whether the background displays correctly.
2. Press Shift + refresh(↻) on Mac or Crtl + refresh(↻) on Windows to see if a hard refresh of the page causes the error to occur.
